tests/expectations/core/test_expect_queried_column_value_frequency_to_meet_threshold.py

```diff
@@ -178,11 +178,9 @@ def test_expect_queried_column_value_frequency_to_meet_threshold_spark(
     warns,
     spark_session,
     basic_spark_df_execution_engine,
+    titanic_df,
 ):
-    df = pd.read_csv(
-        "tests/test_fixtures/configuration_for_testing_v2_v3_migration/data/Titanic.csv"
-    )
-
+    df = titanic_df
```

Comment:
@Shinnnyshinshin How about adding type hints -- for future benefits.  Thanks!

Reply:
thanks for the catch :)